tests/redux/reducers/agencyLanding/agencyLandingReducer-test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * agencyLandingReducer-test.js
+ * Created by Lizzie Salita 7/26/17
+ */
+
+import agencyLandingReducer, { initialState, Agency } from 'redux/reducers/agencyLanding/agencyLandingReducer';
+import Immutable from 'immutable';
+
+describe('agencyLandingReducer', () => {
+    describe('SET_AGENCIES', () => {
+        it('should add Agency objects to the Redux store', () => {
+            let state = agencyLandingReducer(initialState, {});
+
+            const model = new Immutable.OrderedSet([
+                new Agency({
+                        agency_id: 1,
+                        agency_name: 'Test 1',
+                        budget_authority_amount: '1234567',
+                        percentage_of_total_budget_authority: '0.012'
+                    }),
+                    new Agency ({
+                        agency_id: 2,
+                        agency_name: 'Test 2',
+                        budget_authority_amount: '2345678',
+                        percentage_of_total_budget_authority: '0.023'
+                    })
+            ]);
+            const action = {
+              type: 'SET_AGENCIES',
+              agencies: model
```

Comment:
Doesn't the action create the `OrderedSet`? If so, you shouldn't be passing an OrderedSet but an array.